File: codewars/2020/march_12/filtering_even_numbers_8kyu.java
Fix the bug in Filtering method
The method is supposed to remove even numbers from the list and return a list that contains odd number.

However, there is a bug in the method that needs to be resolved.

Instructions
Output
Fix the bug in Filtering method
The method is supposed to remove even numbers from the list and return a list that contains odd number.

However, there is a bug in the method that needs to be resolved.

import java.util.List;
import java.util.*; 
public class Kata13December {
    public static List<Integer> filterOddNumber(List<Integer> listOfNumbers)
    {
      List<Integer> OddList = new ArrayList<>();
        for (int i = 0; i < listOfNumbers.size(); i++)
        {
            if (listOfNumbers.get(i) % 2 != 0)
            {
                OddList.add(listOfNumbers.get(i));
            }
        }
        return OddList;
    }
}

import java.util.*;
import java.lang.*;
import java.util.Collections.*;
import java.util.stream.Collectors;

public class Kata13December {
    public static List<Integer> filterOddNumber(List<Integer> listOfNumbers)
    {
        
        return listOfNumbers.stream().filter(l -> l%2 != 0).collect(Collectors.toList());
    }
}

File: codewars/2020/march_12/filtering_even_numbers_8kyu.cs
Fix the bug in Filtering method
The method is supposed to remove even numbers from the list and return a list that contains odd number.

However, there is a bug in the method that needs to be resolved.

Instructions
Output
Fix the bug in Filtering method
The method is supposed to remove even numbers from the list and return a list that contains odd number.

However, there is a bug in the method that needs to be resolved.

using System;
using System.Collections.Generic;
using System.Linq;
public class Kata
{
  public static List<int> FilterOddNumber(List<int> listOfNumbers)
  {
    List<int> OddList = new List<int>();
    for (int i = 0; i < listOfNumbers.Count; i++)
    {
      if (listOfNumbers[i] % 2 != 0)
      {
        OddList.Add(listOfNumbers[i]);
      }
    }
    return OddList;
  }
}

using System;
using System.Collections.Generic;
using System.Linq;

public class Kata
{
  public static List<int> FilterOddNumber(List<int> numbers)
  {
    return numbers.Where(n => n % 2 == 1).ToList();
  }
}